functorch/.github/workflows/docs.yml

```diff
@@ -1,82 +0,0 @@
-name: Build and Deploy Docs
-on:
-  pull_request:
-    types: [opened, synchronize, reopened]
-  push:
-    branches:
-      - main
```

Comment:
We still need the docs workflow; we have not migrated it over to PyTorch yet

Reply:
Actually, let me think about this for a little more (it depends on what the next PR up in the stack is doing)

Reply:
Well, I don't think it works that way, is it? (Unless this folder is still mirrored to github.com/pytorch/functorch)

Reply:
>  (Unless this folder is still mirrored to github.com/pytorch/functorch)

It is, for now at least